atom/browser/atom_browser_context.cc

```diff
@@ -237,10 +234,8 @@ scoped_refptr<AtomBrowserContext> AtomBrowserContext::From(
     const std::string& partition, bool in_memory,
     const base::DictionaryValue& options) {
   auto browser_context = brightray::BrowserContext::Get(partition, in_memory);
-  if (browser_context)
-    return static_cast<AtomBrowserContext*>(browser_context.get());
-
-  return new AtomBrowserContext(partition, in_memory, options);
+  return (browser_context) ?
+    static_cast<AtomBrowserContext*>(browser_context.get()) :
```

Comment:
For the purpose of readable, I think the original code is better, it was a common early return when cache hits.